File: autopilot/v1/model_build/create-example-1/create-example-1.5.x.rb
# Download the helper library from https://www.twilio.com/docs/ruby/install
require 'rubygems'
require 'twilio-ruby'

# Your Account Sid and Auth Token from twilio.com/console
account_sid = 'ACXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX'
auth_token = 'your_auth_token'
@client = Twilio::REST::Client.new(account_sid, auth_token)

model_build = @client.autopilot
                     .assistants('assistant_unique_name')
                     .model_builds
                     .create(unique_name: 'v0.1')

puts model_build.sid

File: autopilot/v1/assistant/create-example-1/create-example-1.5.x.cs
// Install the C# / .NET helper library from twilio.com/docs/csharp/install

using System;
using System.Collections.Generic;
using Twilio;
using Twilio.Rest.Autopilot.V1;


class Program 
{
    static void Main(string[] args)
    {
        // Find your Account Sid and Token at twilio.com/console
        // DANGER! This is insecure. See http://twil.io/secure
        const string accountSid = "ACXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX";
        const string authToken = "your_auth_token";

        TwilioClient.Init(accountSid, authToken);

        var styleSheet = new Dictionary<string, Object>()
        {
            {"voice", new Dictionary<string, Object>()
                {
                    {"say_voice", "Polly.Joanna"}
                }}
        };

        var assistant = AssistantResource.Create(
            friendlyName: "Styled Assistant",
            uniqueName: "styled-assistant",
            styleSheet: styleSheet
        );

        Console.WriteLine(assistant.Sid);
    }
}
